tests/conftest.py

```diff
@@ -26,17 +32,114 @@ def app_params(request):
             kwargs.update(info.kwargs)
 
     args = [pargs[i] for i in sorted(pargs.keys())]
-    return args, kwargs
+
+    # ##### process pytest.mark.test_params
+
+    if test_params['specific_srcdir'] and 'srcdir' not in kwargs:
```

Comment:
srcdir is used for sphinx project root.
`specific_srcdir` is same but bit different. When set `True` to `specific_srcdir`, the value is automatically converted into test function name.
`test_build_html.py` and `test_intl.py` are using the param.

Why do We need to specify True? We can provide the same capability by setting a unique arbitrary string in `srcdir`. I thought that it is troublesome for the developer to think of such a string, so I provide this function. However, because implicit behavior is also confusing, I will examine whether removing `specific_srcdir` will make development troublesome.